src/routes.js

```diff
@@ -0,0 +1,32 @@
+import VueRouter from "vue-router";
+import BreedsList from "./components/BreedsList.vue";
+
+const router = new VueRouter({
+  mode: "history",
+  base: "/",
+  routes: [
+    {
+      name: "redirect",
+      path: "/",
+      component: BreedsList,
+      redirect: "/breeds/",
+    },
+    {
+      name: "breeds",
+      path: "/breeds/",
+      component: BreedsList,
+    },
+    {
+      name: "breedName",
+      path: "/breeds/:breedName",
+      component: BreedsList,
+      props: true,
+    },
```

Comment:
Dodal sem ta route, vezan na BreedsList komponento, kjer ko uporabnik klikne na `li` znotraj BreedCard komponente=>
`<router-link :to="`/breeds/${breed.name}`">` se mu avtomatsko nastavi dinamični parameter glede na pasmo, ki jo je kliknil.
Znotraj `BreedCard `komponente sem potem na `toggleDetails `metodo dodal ta `scrollIntoView `metodo, da ko uporabnik klikne določeno pasmo, da je aktivna pasma vedno na sredini zaslona.
```
toggleDetails() {
      this.$emit("toggle", this.breed);
      if (this.breed.isActive) {
        this.$refs.activeLi.scrollIntoView({
          behavior: "smooth",
          block: "center",
        });
      }
    },
```
V `BreedsList `komponenti sem dodal `showBreedDetailsOnRouteParam` metodo, s katero če uporabnik ročno vnese url, npr. : `http://127.0.0.1:5173/breeds/akita` se tej aktivni pasmi, v tem primeru Akita, prikažejo details.
`showBreedDetailsOnRouteParam(breedName) {
      if (breedName) {
        this.displayedBreeds.filter((breed) => {
          if (breed.name.toLowerCase() === breedName.toLowerCase()) {
            breed.isActive = true;
          }
        });
      }
    },`
    
Kakor sva govorila bi želel še, da če uporabnik ročno vnese url, da se mu avtomatsko scrolla na vpisano pasmo, oz če uporabnik klikne gumbe nazaj oz naprej v brskalniku,da mu pokaže prejšnjo oziroma naslednjo pasmo

Upam, da je razumljivo 😄